Automattic/jetpack
# Add VIP slug

Fixes #8857

#### Changes proposed in this Pull Request:

* Add VIP as a plan type

#### Testing instructions:

* Flag a Jetpack site as VIP via the RC
* Visit the Jetpack->Settings page and notice upgrade prompts
* Apply patch
* Dashboard should reflect a professional-level plan

Note to the reviewer: I'm weak in react/JS/etc, so I'm not sure if I hit all of the right places for the Plans page to be okay and for the prompts to disappear.
